### PR TITLE
add option packGenerateMakefile

### PR DESCRIPTION
**Pull request by liufengyun**
add option `packGenerateMakefile`

### CODE REVIEW

**Comment by liufengyun** [comment]
BTW, thanks a lot for the wonderful plugin, Dotty is using it for publishing artefacts 👍

**Comment by xerial** [comment]
Thanks. I'll check the test error.

File: src/main/scala/xerial/sbt/pack/PackPlugin.scala [hunk 1]
```diff
@@ -355,14 +357,16 @@ object PackPlugin extends AutoPlugin with PackArchive {
       def linkToScript(name: String) =
         "\t" + """ln -sf "../$(PROG)/current/bin/%s" "$(PREFIX)/bin/%s"""".format(name, name)
 
-      // Create Makefile
-      // TODO Use custom template (packMakefileTemplate)
-      val makefile = {
-        val additionalScripts = binScriptsDir.flatMap(_.listFiles).map(_.getName)
-        val symlink = (mainTable.keys ++ additionalScripts).map(linkToScript).mkString("\n")
-        LaunchScript.generateMakefile(name.value, symlink)
+      if (packGenerateMakefile.value) {
```
Comment (xerial) [inline src/main/scala/xerial/sbt/pack/PackPlugin.scala]: In sbt-1.0, we need to evaluate task outside the conditional branch (e.g., if-else block). 
So could you rewrite this like
```scala
val projectName = name.value
val makefile = { ...
```

Comment (liufengyun) [reply src/main/scala/xerial/sbt/pack/PackPlugin.scala]: Thanks a lot @xerial , I'll push a commit soon.

**Comment by xerial** [comment]
Thanks for the fix. I'll release 0.9.1 soon.

**Comment by xerial** [comment]
Released sbt-pack 0.9.1 with this option.

**Comment by liufengyun** [comment]
Thanks a lot 👍

**Pull request merged**